File: Models/City.cs
using Newtonsoft.Json;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

using GridWebApp.Models.Interfaces;

namespace GridWebApp.Models
{
    [Table("cities")]
    public class City : IModel
    {
        [JsonProperty("CITY_OKATO_ID")]
        [Key]
        public long Id { get; set; }

        [JsonProperty("NAME")]
        public string Name { get; set; }


        [JsonProperty("PARENT_REGION_ID")]
        public int ParentId { get; set; }

        [JsonProperty("PARENT_REGION_NAME")]
        public string ParentName { get; set; }

        [JsonProperty("FO_ID")]
        public int FederalDistrictId { get; set; }
    }
}

File: ClientApp/src/app/_models/city.ts
export class City {
    id: number;
    name: string;
    parent_id: number;
    parent_name: number;
    federal_district_id: number;

    constructor(data: any) {
        this.id = data.id;
        this.name = data.name;
        this.parent_id = data.parentId;
        this.parent_name = data.parentName;
        this.federal_district_id = data.federalDistrictId;
    }
}
